src/formio/base.ts

```diff
@@ -195,3 +195,24 @@ export type InputComponentSchema<
  */
 export interface LayoutComponentSchema<T = never>
   extends Omit<ComponentSchema<T>, UnusedFormioProperties> {}
+
+/**
+ * @group Schema primitives
+ */
+export interface TableViewComponentSchema<T = Record<string, any>>
+  extends Omit<
+      StrictComponentSchema<T[]>,
+      | 'tooltip'
+      | 'description'
+      | 'validate'
+      | 'placeholder'
+      | 'clearOnHide'
+      | 'persistent'
+      | 'hidden'
+      | 'conditional'
+      | 'errors'
+      | 'hideLabel'
+      | 'disabled'
+      | 'validateOn'
```

Comment:
some of these properties are already omitted in `StrictComponentSchema` through `UnusedFormioProperties`, like `persistent`

Is it worthwile to define another base interface, compared to straight up defining the custom type?

Reply:
I thought it might be useful for the children as well that's why I chose to define a new base schema.